iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/LccTest.java

```diff
@@ -58,7 +58,6 @@ public void testBase() {
         assertEquals(1e-5, network.getShuntCompensator("C1_Filter1").getCurrentB(), 0.0);
         assertTrue(network.getShuntCompensator("C1_Filter1").getTerminal().isConnected());
         assertEquals(0.0, network.getShuntCompensator("C1_Filter2").getCurrentB(), 0.0);
-        assertEquals(2e-5, network.getShuntCompensator("C1_Filter2").getMaximumB(), 0.0);
```

Comment:
You should probably keep this test until you remove the method to help sonar/coveralls to accept this PR.